README.md

```diff
@@ -53,66 +79,93 @@ Point tags are key-value pairs (strings) that are associated with a point. Point
 | Resource              | The name and version/alias of Lambda function. (like `DemoLambdaFunc:aliasProd`)           |
 | EventSourceMappings   | AWS Event source mapping Id. (Set in case of Lambda invocation by AWS Poll-Based Services) |
 
-## Standard Metrics
+### Custom Point Tags
+
+You can add custom point tags either while instantiating the agent or inside your handler.
```

Comment:
Also mention specifically that in the below example , we are adding reporter level point tags, meaning these point tags will be reported for all metrics registered with the agent.

Reply:
Good idea! I've added that